COMPONENT_GUIDE.md

```diff
@@ -252,3 +252,39 @@ Text
 
 </Caution>
 ```
+
+## Related Resources
+
+The related resources component is controlled by specific Frontmatter slugs that
+are defined on a page.
+
+By setting the Frontmatter for `resources` and `tags` you can control what is populated
+in this component. Please review the [Style Guide](STYLE_GUIDE) for further details
```

Comment:
It looks like this `Style Guide` link 404s. Perhaps you need a `.md` extension on the end of this?